client/resource_group/controller/limiter.go

```diff
@@ -370,6 +370,10 @@ func (lim *Limiter) reserveN(now time.Time, n float64, maxFutureReserve time.Dur
 		lim.maybeNotify()
 	} else {
 		lim.last = last
+		if lim.limit == 0 {
```

Comment:
why need `limit == 0`?

Reply:
When fillrate is greater than 0, it is in trickle state or degraded mode. If you are in trickle state, the server is allocating tokens as much as possible. You don't need to take tokens.